treccovid_semantic_search/README.md

```diff
@@ -0,0 +1,261 @@
+## Trec-Covid Semantic Search workload
+
+This workload uses OpenSearch pretrained model and ml-common-plugin to embed vectors. It is based on the neural search tutorial https://opensearch.org/docs/latest/search-plugins/neural-search-tutorial/ 
```

Comment:
What are the main metrics you want to get with this workload? You are not checking correctness of the neural query results, is it mainly query latency?

Reply:
The main metrics are throughput and latency. No checking correctness, I will try to add this later.